CHANGELOG.md

```diff
@@ -9,11 +9,13 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### Oneview Features supported
 - Enclosure
 - Enclosure Group
-- Interconnects
+- Hypervisor Manager
+- Interconnect
```

Comment:
Please remove this as it is not related to our change

Reply:
Since we are mentioning all the resource names in singular form, I made this change.

Reply:
okay